mod/Jailbreak.Warden/Commands/WardenPeaceCommandsBehavior.cs

```diff
@@ -0,0 +1,42 @@
+﻿using CounterStrikeSharp.API.Core.Attributes.Registration;
+using CounterStrikeSharp.API.Core;
+using CounterStrikeSharp.API.Modules.Commands;
+using Jailbreak.Public.Behaviors;
+using Jailbreak.Public.Mod.Warden;
+using Jailbreak.Public.Generic;
+
+namespace Jailbreak.Warden.Commands;
+
+public class WardenPeaceCommandsBehavior : IPluginBehavior
+{
+
+    private readonly IWardenPeaceService _peaceService;
+    private readonly ICoroutines _coroutines;
+
+    private static readonly float _muteTime = 10.0f;
+
+    public WardenPeaceCommandsBehavior(IWardenPeaceService peaceService, ICoroutines coroutines)
+    {
+        _peaceService = peaceService;
+        _coroutines = coroutines;
+    }
+
+    [ConsoleCommand("css_peace", "Gives everybody some peace of mind by muting Prisoners/Guards for x seconds (warden is exempt from this).")]
+    [CommandHelper(0, "", CommandUsage.CLIENT_ONLY)]
+    public void Command_Peace(CCSPlayerController? invoker, CommandInfo command)
+    {
+
+        if (invoker == null) return;
+
+        CCSPlayerController? warden = _peaceService.GetWarden();
+
+        if (warden == null) return;
+
+        // we only want the warden to be able to run this!
+        if (!invoker.Equals(warden)) return;
```

Comment:
Not a fan of wrapping around already written logic for checking if activator is warden. Please use https://github.com/edgegamers/Jailbreak/blob/main/public/Jailbreak.Public/Mod/Warden/IWardenService.cs#L15

Reply:
Also done